docs/2_configuration/preconfiguration.md

```diff
@@ -1,6 +1,13 @@
-# Pre-configuring the FDM Monster Server
-
-> :warning: **After each environment change the FDM Monster server must be restarted for the change to take effect!**
+---
+layout: default
+title: Pre-configuring the FDM Monster Server
+parent: Configuration
+nav_order: 1
```

Comment:
`sidebar_position` does exist through the docs plugin that we have by default

Reply:
Todo when working on section 2 and above will be handled.